Service/src/main/java/Users/User.java

```diff
@@ -0,0 +1,5 @@
+package Users;
+
+public class User {
```

Comment:
Oh, I thought that my User class was abstract and I inherited the Admin and Client classes from it. I didn't notice this

Reply:
I thought about abstract class because we can divide users into two groups: clients and admins. We can't create object of User class. User class just describes common functional for inherited classes.